changelogs/fragments/2604_fix_regression_iptables-nft.yml

```diff
@@ -0,0 +1,5 @@
+---
+bugfixes:
```

Comment:
Sorry that I didn't realize this earlier, but since this fixes something that has not yet been released, this shouldn't have a changelog fragment. Can you please remove it?

Reply:
yes, I can :)